Feature/HM-37-Property NumberOfRooms becoming a string field

petarbelchev opened the pull request:
The NumberOfRoom property of the Property becomes a string field.

**The migration needs to be executed and tested!**

petarbelchev committed:
The NumberOfRoom property of the Property becomes a string field.

petarbelchev committed:
Unused Model Builder method has been removed from the ChangedNumberOfRoomsToString migration.

rosteslav committed:
-get latest

rosteslav committed:
- migrations update

rosteslav commented on src/Properties/Properties.Domain/Entities/Property.cs:
```diff
@@ -3,7 +3,7 @@ namespace BuildingMarket.Properties.Domain.Entities
     {
         public int Id { get; set; }
-        public required int NumberOfRooms { get; set; }
+        public required string NumberOfRooms { get; set; }
```
You should change the entity in all the other projects that refer it, that is in Images Microservice

rosteslav commented on src/Properties/Properties.Infrastructure/Persistence/PropertiesContext.cs:
On OnModelCreating should set 
                    .HasColumnType("character varying")
                    .HasMaxLength(255);

This is a string now. Should update in Images microservice as well

petarbelchev committed:
The NumberOfRooms property of the Property has been changed to a string type in other projects.
A column type and maximum length have been added to the NumberOfRooms property in the database context, along with the migration setup.

petarbelchev committed:
The Images table has been excluded from migrations in the PropertiesContext.
